src/constants/actionTypes.js

```diff
@@ -0,0 +1,5 @@
+import keyMirror from 'keymirror';
+
+export default keyMirror({
+  SET_NAME: null
```

Comment:
If it pleases the court, I'd say we use something other than `null` here, assuming keymirror supports it. Maybe `0` or `1` ?

Reply:
I'm fine with that. I think it'll work either way. I'm pretty sure it just throws away the value, and uses the key as the value instead.

Reply:
So I looked at the `keymirror` source code and I think you're both right that it doesn't really matter what the value is here since it just overwrites it when the key. With that being said I'm going to leave it as `null` in the boilerplate since all the documentation around `keymirror` shows it as `null` and leave it up to the project team to change it if they want. I'd be curious to know why the preference for `0` or `1` though?